src/s2geography/geoarrow.h

```diff
@@ -65,6 +65,21 @@ class Reader {
   std::unique_ptr<ReaderImpl> impl_;
 };
 
+class WriterImpl;
+
+class Writer {
+  public:
+  enum class OutputType { kWKT, kWKB };
```

Comment:
I did the same in my draft, but for this case was thinking if a single top-level enum would make sense? Although of course for export you might want to say something like "points, interleaved" which then doesn't make sense for import .. (unless we make the "interleaved" part a separate option in ExportOptions)

Reply:
> For your purposes I think requiring an `ArrowSchema*` if you want fancy geoarrow export is probably OK since you can generate that from Python?

At the moment we don't have python code in spherely (at some point we might want to add some python code anyhow, but right now everything is in pybind11 C++ code, which can call python code of course (like cython), but not that convenient)

Of course we could require the user to provide a type (extracting the ArrowSchema through `__arrow_c_schema__`), but I think for a basic "geoarrow" export (with type inferred from the geographies), it would be nice to provide that out of the box, I think